src/js/health-records/containers/Main.jsx

```diff
@@ -225,35 +226,9 @@ export class Main extends React.Component {
                 !datePickerDisabled && !invalidFormatError && missingDateError && <p className="date-range-error">
                   {validationErrorMessages.missingDateError}</p>
               }
-              <span>Custom date range</span>
-              <div className="date-range-fields">
-                <DatePicker
-                  aria-label="Custom date range: start date"
-                  id="custom-date-start"
-                  onBlur={handleFormattedDate()}
-                  onChange={this.handleStartDateChange}
-                  onFocus={() => this.setState({ startDateError: null })}
-                  placeholderText="MM/DD/YYYY"
-                  selected={startDate}
-                  disabled={datePickerDisabled}
-                  maxDate={endDate}
-                  tetherConstraints={[{ to: 'scrollParent', attachment: 'none' }]}
-                  className={!datePickerDisabled && this.state.startDateError ? 'date-range-error' : ''}/>
-                <span>&nbsp;to&nbsp;</span>
-                <DatePicker
-                  aria-label="Custom date range: end date"
-                  id="custom-date-end"
-                  onBlur={handleFormattedDate(false)}
-                  onChange={this.handleEndDateChange}
-                  onFocus={() => this.setState({ endDateError: null })}
-                  placeholderText="MM/DD/YYYY"
-                  selected={endDate}
-                  disabled={datePickerDisabled}
-                  minDate={startDate}
-                  maxDate={moment()}
-                  tetherConstraints={[{ to: 'scrollParent', attachment: 'none' }]}
-                  className={!datePickerDisabled && this.state.endDateError ? 'date-range-error' : ''}/>
-              </div>
+              <span id="vets-custom-date-range">Custom date range</span>
+              {/* The span class="usa-sr-only" is invisible to sighted users, but will offer additional instruction for screen readers */}
```

Comment:
Is this comment necessary? I think it should be evident already from the class name, even if it needs to be looked up.